test/src/test/java/hudson/util/RobustReflectionConverterTest.java

```diff
@@ -242,7 +236,7 @@ public void testRestInterfaceFailure() throws Exception {
             r.jenkins.setSecurityRealm(r.createDummySecurityRealm());
             WebClient wc = r.createWebClient()
                     .withThrowExceptionOnFailingStatusCode(false);
-            wc.withBasicApiToken(test);
+            wc.withBasicApiToken("test");
```

Comment:
Took me a while to track this down. The original test was creating a `User` object, then `reload`ing, rendering that object stale, but the legacy API token still worked because it is always regenerated the same way. When using new-style API tokens, the first block saves a token to disk, but the second block adds to the token list for the old `User` object, whereas the web filter is checking against what it loaded from disk, which only includes the original token.